src/main/java/core/org/akaza/openclinica/domain/xform/dto/Head.java

```diff
@@ -4,6 +4,7 @@
 
     private String title;
     private Model model;
+    private String crossform_reference;
```

Comment:
looks like it is called crossform_references (plural) in form-service. See https://github.com/OpenClinica/form-service/pull/82/files